### DISCUSSION TITLE
Placing imgs IN cells

### DISCUSSION THREAD

**Discussion by ReniIrinyi** [exceljs/exceljs#2334] [Q&A]
I use the following code to save the signatures. The library is great, and the code works, but I can't get the image to "grow" with the cell, or at least to be placed in the cell.  In any case, it will be placed on the cell. I'm a beginner, maybe I'm missing something, or is there some prop I don't know about? Thanks in advance for your help! 

`async function addImageToCell(
  dataRow,
  imageBuffer,
  columnIndex,
  workbook,
  sheet
) {
  try {
    const imageId = workbook.addImage({
      buffer: imageBuffer,
      extension: "png",
    });
    sheet.addImage(imageId, {
      tl: { col: columnIndex, row: dataRow.number },
      ext: { width: 120, height: 120 },
      editAs: "oneCell",
    });
  } catch (error) {
    console.error("Error adding image to cell:", error);
    return;
  }
}`

**Comment by hshoja** [comment]
I guess you mean having this option in excel 
<img width="382" alt="Screenshot 2023-12-18 at 16 03 50" src="https://github.com/exceljs/exceljs/assets/14098200/2f294e94-c7ea-4c2d-abe0-396b6eea8c90">

I'm also curious if the library provides support for it?

**Reply by ReniIrinyi** [reply]
I already found out🙋‍♀️😊

**Reply by hshoja** [reply]
so you might want to share it 😊 ?
